bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/protobuf/BigtableZeroCopyByteStringUtil.java

```diff
@@ -12,6 +12,29 @@
  * @version $Id: $Id
  */
 public final class BigtableZeroCopyByteStringUtil {
+
```

Comment:
Ok, tracing this further looks like we can get rid of our .wrap() method here and use UnsafeByteOperations. 

But the ByteString class is still missing a way to go the other direction (i.e. "unwrap") AFAICT.